kube/stateful_set.go

```diff
@@ -68,7 +68,12 @@ func getVolumeClaims(role *model.Role, createHelmChart bool) []helm.Node {
 		}
 
 		meta := helm.NewMapping("name", volume.Tag)
-		meta.Add("annotations", helm.NewMapping(VolumeStorageClassAnnotation, storageClass))
+		annotationList := helm.NewMapping()
+		annotationList.Add(VolumeStorageClassAnnotation, storageClass)
```

Comment:
(Technically, this can be `annotationList := helm.NewMapping(VolumeStorageClassAnnotation, storageClass)` but that really isn't worth your time or mine)